opencensus/stats/view_data.py

```diff
@@ -72,13 +73,17 @@ def end(self):
 
     def get_tag_map(self, context):
         """function to return the tag map based on the context"""
-        if context.items() <= self.tag_map.items():
-            return self.tag_map
-        else:
-            tags = self.tag_map
-            for tag_key, tag_value in context.items():
-                tags[tag_key] = tag_value
-            return tags
+        try:
+            if context.items() <= self.tag_map.items():
+                return self.tag_map
+            else:
+                tags = self.tag_map
+                for tag_key, tag_value in context.items():
+                    tags[tag_key] = tag_value
+                return tags
+        except ValueError:  # pragma: NO COVER
```

Comment:
Suggestion: is it possible to check the type of `self.tag_map` instead of doing the try-except logic?

Reply:
We can check is self.tag_map is None with if/else, should I do that instead?